### DISCUSSION TITLE
Forward Spotify Album Picture from Chromecast Speaker to Home Assistant Add-on even when Playback wasn't started in Music Assistant

### DISCUSSION THREAD

**Discussion by jamstruth** [music-assistant/support#3299] [Feature requests and ideas]
I'm using Music Assistant alongside my Chromecast speakers. Its great but I don't always use Music Assistant to start playback on my devices, sometimes I use the Spotify App to start playback out of convenience.

In Home Assistant the Music-Assistant items correctly show playing and basic playback controls for the device but it doesn't show the album picture
![image](https://github.com/user-attachments/assets/102bd6fd-4e1a-4bda-b4f9-bdbb758f0f65)

If Music Assistant started playback then full controls are shown and the picture. I understand this may be a limitation of what can be achieved from information from Chromecast but it would be nice to maintain the picture for aesthetics. The UI shows the picture fine

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/3208
